astropy/io/fits/file.py

```diff
@@ -157,7 +151,7 @@ def __init__(self, fileobj=None, mode=None, memmap=None, overwrite=False,
             mode = 'readonly'
 
         # Handle raw URLs
-        if (isinstance(fileobj, str) and
+        if (isinstance(fileobj, (str, bytes)) and
```

Comment:
Is this a bug fix?

Reply:
Ah, I see the comments above. This is for Python 3.6. I guess strictly speaking, it doesn't need to be backported.